deployment-templates/deploy-forseti.yaml.sample

```diff
@@ -55,7 +54,8 @@ resources:
     scanner-bucket: SCANNER_BUCKET
     database-name: forseti_security
     organization-id: YOUR_ORG_ID
-    src-path: https://github.com/GoogleCloudPlatform/forseti-security/archive/v1.0.tar.gz
+    release-version: "master"
```

Comment:
nit: I would rather think that the default should point at a stable release version, with getting the bleeding edge "master" being an option.